### ISSUE TITLE
Versions view broken

### ISSUE THREAD

**Issue by mrplow87** [ytti/oxidized-web#208]
After updating Oxidized(-web|-script) on Debian 9 and in a clean install on a fresh CentOS 8, clicking the version button in the list of hosts results in the following error:

Example URL: `https://192.0.2.1/node/version?node_full=GroupName/NodeName`

```
2020-05-15 15:38:34 - ArgumentError - wrong number of arguments (given 0, expected 1+):
	/usr/local/share/gems/gems/oxidized-web-0.13.1/lib/oxidized/web/views/versions.haml:24:in `count'
	/usr/local/share/gems/gems/oxidized-web-0.13.1/lib/oxidized/web/views/versions.haml:24:in `__tilt_47242120843340'
	/usr/local/share/gems/gems/tilt-2.0.10/lib/tilt/template.rb:170:in `call'
	/usr/local/share/gems/gems/tilt-2.0.10/lib/tilt/template.rb:170:in `evaluate'
	/usr/local/share/gems/gems/tilt-2.0.10/lib/tilt/haml.rb:24:in `evaluate'
	/usr/local/share/gems/gems/tilt-2.0.10/lib/tilt/template.rb:109:in `render'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:823:in `render'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:677:in `haml'
	/usr/local/share/gems/gems/oxidized-web-0.13.1/lib/oxidized/web/webapp.rb:241:in `out'
	/usr/local/share/gems/gems/oxidized-web-0.13.1/lib/oxidized/web/webapp.rb:165:in `block in <class:WebApp>'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1611:in `call'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1611:in `block in compile!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:975:in `block (3 levels) in route!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:994:in `route_eval'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:975:in `block (2 levels) in route!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1015:in `block in process_route'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1013:in `catch'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1013:in `process_route'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:973:in `block in route!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:972:in `each'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:972:in `route!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1085:in `block in dispatch!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1067:in `block in invoke'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1067:in `catch'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1067:in `invoke'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1082:in `dispatch!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:907:in `block in call!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1067:in `block in invoke'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1067:in `catch'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1067:in `invoke'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:907:in `call!'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:895:in `call'
	/usr/local/share/gems/gems/rack-protection-1.5.5/lib/rack/protection/xss_header.rb:18:in `call'
	/usr/local/share/gems/gems/rack-protection-1.5.5/lib/rack/protection/path_traversal.rb:16:in `call'
	/usr/local/share/gems/gems/rack-protection-1.5.5/lib/rack/protection/json_csrf.rb:18:in `call'
	/usr/local/share/gems/gems/rack-protection-1.5.5/lib/rack/protection/base.rb:49:in `call'
	/usr/local/share/gems/gems/rack-protection-1.5.5/lib/rack/protection/base.rb:49:in `call'
	/usr/local/share/gems/gems/rack-protection-1.5.5/lib/rack/protection/frame_options.rb:31:in `call'
	/usr/local/share/gems/gems/rack-1.6.13/lib/rack/nulllogger.rb:9:in `call'
	/usr/local/share/gems/gems/rack-1.6.13/lib/rack/head.rb:13:in `call'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/show_exceptions.rb:25:in `call'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:182:in `call'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:2013:in `call'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1487:in `block in call'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1787:in `synchronize'
	/usr/local/share/gems/gems/sinatra-1.4.8/lib/sinatra/base.rb:1487:in `call'
	/usr/local/share/gems/gems/rack-1.6.13/lib/rack/urlmap.rb:66:in `block in call'
	/usr/local/share/gems/gems/rack-1.6.13/lib/rack/urlmap.rb:50:in `each'
	/usr/local/share/gems/gems/rack-1.6.13/lib/rack/urlmap.rb:50:in `call'
	/usr/local/share/gems/gems/rack-1.6.13/lib/rack/builder.rb:153:in `call'
	/usr/local/share/gems/gems/puma-3.11.4/lib/puma/configuration.rb:225:in `call'
	/usr/local/share/gems/gems/puma-3.11.4/lib/puma/server.rb:632:in `handle_request'
	/usr/local/share/gems/gems/puma-3.11.4/lib/puma/server.rb:446:in `process_client'
	/usr/local/share/gems/gems/puma-3.11.4/lib/puma/server.rb:306:in `block in run'
	/usr/local/share/gems/gems/puma-3.11.4/lib/puma/thread_pool.rb:120:in `block in spawn_thread'
```

I've made sure that the node I'm trying to view has had an configuration update before trying to view the versions.

I've tried past oxidized-web versions (current, 0.12.1, 0.11.1) or disabling groups, but everything results in the same error.

**Comment by mrplow87** [comment]
adding `&format=json` to the URL results in `"node not found"`

**Comment by wk** [comment]
This has been resolved in version 0.28.0, in [this](https://github.com/ytti/oxidized/commit/d96a592be605149f8a4f8d45e8b35e1ed8f3049c) commit. Closing this issue as while valid, it's already been addressed on the ytti/oxidized#2049 and ytti/oxidized#2053 and a fix is available.
